src/components/pages/CaseOverview/EditCaseDetails.js

```diff
@@ -230,12 +230,12 @@ const EditCaseDetails = props => {
           />
         </Form.Item>
         <Checkbox
-          id="credible"
-          name="credible"
-          value={formValues.credible}
+          id="credibility"
+          name="credibility"
+          value={formValues.credibility}
           onClick={onCheck}
         >
-          Credible
+          credibility
```

Comment:
Will was pointing this out, but I thought to add a comment. You will more than likely want to capitalize credibility here on 238

Reply:
Revision pushed.